assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -82,12 +82,9 @@ const CartLineItemRow = ( { lineItem } ) => {
 		},
 	} = lineItem;
 
-	const {
-		quantity,
-		changeQuantity,
-		removeItem,
-		isPending: itemQuantityDisabled,
-	} = useStoreCartItemQuantity( lineItem );
+	const { quantity, changeQuantity, removeItem } = useStoreCartItemQuantity(
+		lineItem
+	);
```

Comment:
what's the rationale for no longer exposing an `isPending` property? Do we not want to disable the quantity input?

Reply:
`isPending` is now `isPendingQuantity` and it tells us if the quantity update has been dispatched or not, we don’t need that.
previously `isPending` was actually `isPendingDelete` but since we're now deleting them immediately, it is not needed and was not implemented.

Reply:
So there's a few things that have happened here then.

- the logic for tracking and acting on whether an item is _being removed_ (the former pending quantity logic) has been deleted.
- this pull _adds_ (via repurposing existing logic) for tracking quantity changes and using that internally in the hook to increment/decrement the calculating count.
- this pull implemented removing items instantly from the ui instead of waiting for the server response that it was successfully removed which negated the need to disable quantity selection field on removal.

Is that an accurate description of what is happening in this pull?  If so, I think you could do either of the following:

-  finish #2126 which takes care of the item removing bit (and can disable any inputs along with applying opacity while an item is being removed).  You'll still need a flag for the pending removal resolution state (and you can apply the renames etc in the pull for that issue).
- once that is done, you can do this pull which will _add_ pending resolution state for quantities that you are able to use for tracking increment/decrementing the calculating counts.

OR

- implement resolution state tracking for pending removal in this pull (along with the new pending quantity resolution state) and use the pending removal resolution state to disable quantity select fields (so there's no regression with existing work).
- work on #2126 after this pull gets merged to add opacity on the removal of items (using the new resolution state added here).

If it was up to me, I'd probably take the second approach.

Reply:
I'm working on the second already and will push it now (applying styles to the item while loading).

Reply:
this is fixed in a7ba54c45e9bd3807409448120e0e9890457ace5

Reply:
I went with not disabling them using that flag, but by using `pointer-events: none` on the parent.